bundle-size/test/fixtures/requested_reviewers.json

```diff
@@ -0,0 +1,27 @@
+{
+  "users": [
+    {
+      "login": "jridgewell",
+      "id": 112982,
+      "node_id": "MDQ6VXNlcjExMjk4Mg==",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/112982?v=4",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/jridgewell",
+      "html_url": "https://github.com/jridgewell",
+      "followers_url": "https://api.github.com/users/jridgewell/followers",
+      "following_url": "https://api.github.com/users/jridgewell/following{/other_user}",
+      "gists_url": "https://api.github.com/users/jridgewell/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/jridgewell/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/jridgewell/subscriptions",
+      "organizations_url": "https://api.github.com/users/jridgewell/orgs",
+      "repos_url": "https://api.github.com/users/jridgewell/repos",
+      "events_url": "https://api.github.com/users/jridgewell/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/jridgewell/received_events",
+      "type": "User",
+      "site_admin": false
```

Comment:
Are all these fields necessary? Consider dropping them assuming it's possible.

Reply:
This is just a copy of how a real response would look like. For tests, I'd rather use a complete real fixture than a mock one